.github/actions/build_setup/action.yml

```diff
@@ -0,0 +1,33 @@
+name: "Build Setup"
+description: "Set up building environment"
+inputs:
+  github_token:
+    description: "Github Token"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Configure Cargo
+      shell: bash
+      run: |
+        cp ./.cargo/ci.toml $HOME/.cargo/config.toml
+
+    - name: Ubuntu install clang and library
+      if: ${{matrix.config.os == 'ubuntu-latest'}}
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y clang clang-tools libclang-dev llvm
+
+    - uses: Warchant/setup-mold@v1
```

Comment:
I believe it's a bug in `Warchant/setup-mold`.

`Warchant/setup-mold` handles `default` wrongly in `v1` which will replace the default `ld` to clang instead. This has been fixed in https://github.com/Warchant/setup-mold/commit/c301b9255e7860fef789d6830d562dd99497b437

Reply:
But I [did](https://github.com/datafuselabs/databend/runs/5257864443?check_suite_focus=true#step:3:98) install `clang` in the system. 
`aarch64` target was built by docker container `datafuselabs/aarch64-buildtools`, and it did configured  its own environment variables like gnu linker used in linking phase.
And no less than above I used clang only for x64 target, as written in databend/.cargo/ci.toml
I can still hardly figure out exactly why my PR crashed...
After all, I'll try to fix it by adapting the version.